velox/connectors/hive/storage_adapters/s3fs/tests/S3UtilTest.cpp

```diff
@@ -20,16 +20,51 @@
 
 using namespace facebook::velox;
 
+// TODO: Each and every single one prefix S3, S3A, OSS implementation
```

Comment:
Combine both into
`// TODO: Each prefix should be implemented as its own filesystem.` ?

Reply:
Fixed.